doc/cqueues.tex

```diff
@@ -434,6 +434,9 @@ \subsubsection{\routine{cqueues.COMMIT}}
 \subsubsection[\fn{socket.listen}]{\fn{socket.listen\{ $\ldots$ \}}}
 	Like \fn{socket.listen} with list arguments, but takes a table of named arguments. See also \fn{socket.connect\{\}}.
 
+\subsubsection[\fn{socket.fdopen}]{\fn{socket.fdopen(FD number)}}
+	Open an existing bound file-descriptor.  That is useful e.g. for listening on sockets passed from systemd.
```

Comment:
No need for it to be bound

Reply:
OK.  I knew I had only vague notion about the constrains of the function.